samples/snippets/create_training_pipeline_image_classification_sample.py

```diff
@@ -30,14 +30,13 @@ def create_training_pipeline_image_classification_sample(
     # Initialize client that will be used to create and send requests.
     # This client only needs to be created once, and can be reused for multiple requests.
     client = aiplatform.gapic.PipelineServiceClient(client_options=client_options)
-
-    icn_training_inputs = trainingjob.definition.AutoMlImageClassificationInputs(
+    training_task_inputs_object = trainingjob.definition.AutoMlImageClassificationInputs(
         multi_label=True,
-        model_type=trainingjob.definition.AutoMlImageClassificationInputs.ModelType.CLOUD,
+        model_type="CLOUD",
```

Comment:
This will be fixed in a follow up PR.  Tracking issue: https://github.com/googleapis/python-aiplatform/issues/143